.github/workflows/fetch-and-ingest-genbank-master.yml

```diff
@@ -99,6 +113,7 @@ jobs:
         AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
         PAT_GITHUB_DISPATCH: ${{ secrets.GH_TOKEN_NEXTSTRAIN_BOT_WORKFLOW_DISPATCH }}
         TRIAL_NAME: ${{ inputs.trial_name }}
+        FETCH_FROM_DATABASE: ${{ inputs.fetch_from_database != '' && inputs.fetch_from_database || true }}
```

Comment:
This, combined with the `type: boolean` of `inputs.fetch_from_database`, is going to result in `FETCH_FROM_DATABASE` being either the strings `true` or `false`, right?

And then the Bash condition (earlier up) is `[[ "$FETCH_FROM_DATABASE" ]]`, which evaluates to true (exits 0) for both `[[ "true" ]]` and `[[ "false" ]]`. Yeah?

So does setting `inputs.fetch_from_database` to `false` actually work? (or am I missing something?)

Reply:
Oh, I see it gets fixed in a later commit, but also https://github.com/nextstrain/ncov-ingest/pull/440#discussion_r1571480694.